tests/meta.rs

```diff
@@ -115,7 +115,7 @@ fn size() {
     assert_size!(Date, 4, 8);
     assert_size!(Duration, 16, 16);
     assert_size!(OffsetDateTime, 16, 16);
-    assert_size!(PrimitiveDateTime, 12, 12);
+    assert_size!(PrimitiveDateTime, 16, 16);
```

Comment:
This is far from ideal, and is directly caused by the difference in alignment. How much of a difference does the alignment make?

Reply:
Cf #599